src/renderer/utils/blocks.ts

```diff
@@ -78,8 +67,9 @@ export const sendBlock = async (
 
   const {ip_address: ipAddress, port, protocol} = activeBankConfig;
   const address = formatAddress(ipAddress, port, protocol);
-
-  const block = await createBlock(primaryValidatorConfig, senderSigningKey, senderAccountNumber, txs);
+  const sender = new Account(senderSigningKey);
+  const balanceLock = await fetchAccountBalanceLock(senderAccountNumber, primaryValidatorConfig);
+  const block = sender.createBlockMessage(balanceLock, txs);
   await axios.post(`${address}/blocks`, block, {
```

Comment:
The post requests here will be handled by sdk in later fix right ?
Or how is it duppose to work; I believe all interactions with chain will go through us eventually

Reply:
It would be handled when I implement the payment handler class